ledger/participant-integration-api/src/main/scala/platform/store/dao/events/PostCommitValidation.scala

```diff
@@ -16,7 +17,7 @@ import com.daml.ledger.participant.state.v1.{CommittedTransaction, RejectionReas
   *
   * Post-commit validation is relevant for three reasons:
   * - keys can be referenced by two concurrent interpretations, potentially leading to
-  *   either create nodes with duplicate active keys or lookup-by-key nodes referring to
+  * either create nodes with duplicate active keys or lookup-by-key nodes referring to
```

Comment:
I'm not sure if you intended this change.

Reply:
Nope, thanks for pointing it out. Autoformat strikes again.

Reply:
```suggestion
  *   either create nodes with duplicate active keys or lookup-by-key nodes referring to
```